### PR TITLE
Add instructions for upgrading Windows integrations

### PR DESCRIPTION
**Pull request by rhetoric101**
We had an old CSAT comment asking why we didn't have upgrade steps for Windows integrations.

### CODE REVIEW

File: src/content/docs/infrastructure/host-integrations/installation/update-infrastructure-host-integration-package.mdx [hunk 1]
```diff
@@ -112,3 +112,45 @@ To update the on-host integration using Zypper:
...
+
+## Update Microsoft Windows integrations [#windows]
+
+Here are two approaches, depending on the integration you're using.
+
+### Microsoft Windows services integration [#ms-services]
+
+The Microsoft Windows services integration is bundled with the New Relic infrastructure agent, so to update the integration, you need to upgrade the infrastructure agent. See [Update the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/update-or-uninstall/update-infrastructure-agent).
+
+### Microsoft SQL Server monitoring integration [#ms-sqlserver]
```
Comment (paologallinaharbur) [inline src/content/docs/infrastructure/host-integrations/installation/update-infrastructure-host-integration-package.mdx:124]: There are [multiple integrations](http://nr-downloads-ohai-staging.s3-website-us-east-1.amazonaws.com/infrastructure_agent/windows/integrations/) for windows. 

We(coreint) support:
```
nri-apache/                                                                      -
nri-cassandra/                                                                   -
nri-consul/                                                                      -
nri-couchbase/                                                                   -
nri-elasticsearch/                                                               -
nri-f5/                                                                          -
nri-haproxy/                                                                     -
nri-kafka/                                                                       -
nri-memcached/                                                                   -
nri-mongodb/                                                                     -
nri-mssql/                                                                       -
nri-mysql/                                                                       -
nri-nagios/                                                                      -
nri-nginx/                                                                       -
nri-postgresql/                                                                  -
nri-rabbitmq/                                                                    -
nri-redis/                                                                       -
nri-varnish/                                                                     -
nri-vsphere/                                                                     -
```

Caos supports, however the upgrade process should be exactly the same
```
nri-jmx/                                                                         -
nri-snmp/                                                                        -
nrjmx/    
```

I am not sure about `nri-winpkg`

File: src/content/docs/infrastructure/host-integrations/installation/update-infrastructure-host-integration-package.mdx [hunk 2]
```diff
@@ -69,46 +92,133 @@ Use either of these options to check whether your infrastructure on-host integra
...
+
+    You can update integrations for services running on Windows Server with our MSI installer or the official zip files. We recommend the MSI installation.
+
+    1. Download the latest MSI installer image [from our repository](http://download.newrelic.com/infrastructure_agent/windows/integrations/).
+    2. In an admin account, run the install script using an absolute path.
+
+      ```
+      msiexec.exe /qn /i <var>PATH\TO\INSERT_THE_INTEGRATION_NAME.msi</var>
+      ```
+    3. Change `C:\Program Files\New Relic\newrelic-infra\integrations.d\integration-name-config.yml.sample` to `integration-name-config.yml`, and edit according to your needs.
```
Comment (carlossscastro) [inline src/content/docs/infrastructure/host-integrations/installation/update-infrastructure-host-integration-package.mdx:133]: This step 3 can be removed. This is only needed for a first install. When updating the installer will keep the existing `integration-name-config.yml` file

File: src/content/docs/infrastructure/host-integrations/installation/update-infrastructure-host-integration-package.mdx [hunk 3]
```diff
@@ -69,46 +92,133 @@ Use either of these options to check whether your infrastructure on-host integra
```
Comment (carlossscastro) [inline src/content/docs/infrastructure/host-integrations/installation/update-infrastructure-host-integration-package.mdx:92]: On this section we can add the command to check the package version on a windows server.


On a PowerShell window run the following command to check the version installed:

```
get-package -name "*INTEGRATION_FILE_NAME*" | select name, version
```

mssql integration for example:
```
get-package -name "*nri-mssql*" | select name, version
```

File: src/content/docs/infrastructure/host-integrations/installation/update-infrastructure-host-integration-package.mdx [hunk 4]
```diff
@@ -69,46 +92,133 @@ Use either of these options to check whether your infrastructure on-host integra
 
 To find the file name for a specific integration, see the [integration documentation](/docs/infrastructure/host-integrations).
 
-## Update using apt (Debian or Ubuntu) [#apt]
+## Pick your update method based on your environment [#install]
+
+Your services may be running on a single physical host, on a VM, or in a container orchestrated by Kubernetes or ECS. Whatever the setup, our on-host integrations can adapt to your environment and send data to New Relic; all you have to do is choose the appropriate update method.
+
+### You're running services in orchestrated environments [#orchestrated]
```
Comment (carlossscastro) [inline src/content/docs/infrastructure/host-integrations/installation/update-infrastructure-host-integration-package.mdx:99]: We have no support for windows integrations in orchestrated environments so no further info is needed here.

For linux the links are OK as upgrading is as a new install so they are relevant here.

**Review by carlossscastro — CHANGES_REQUESTED** [review]

**Review by barbnewrelic — APPROVED** [review]
This is impressive! Succinct yet comprehensive. Thanks for all your hard work on this!

**Pull request merged**